pkgs/tools/nix/nixedit/default.nix

```diff
@@ -0,0 +1,60 @@
+{
+  pkgs ? import <nixpkgs> { },
+}:
+
+pkgs.stdenv.mkDerivation {
+  pname = "nixedit";
+  version = "1.0.0";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "fndov";
+    repo = "nixedit";
+    rev = "main";
```

Comment:
`main` is a branch, that means that building this derivation next time, it might fail since the hash will be invalid. You need to fix the src using a commit ID, see an example at https://github.com/NixOS/nixpkgs/pull/348180/files